src/system/startup.c

```diff
@@ -40,10 +40,10 @@ __attribute__((constructor(101))) static void pros_init(void) {
 
 	vdml_initialize();
 
-	system_daemon_initialize();
-
 	display_initialize();
 
+	system_daemon_initialize();
```

Comment:
Can you link to this issue or throw an explanation comment here for the re-ordering?

Reply:
explained in this very PR! 😀 

https://github.com/purduesigbots/pros/pull/144/files/e5783e32cc447bd0542e5e27baa3376e90c87234#issuecomment-496901942

Reply:
explained in this very PR! 😁 

https://github.com/purduesigbots/pros/pull/144/#issuecomment-496901942

Reply:
Right, sorry for not being more clear. I'm thinking it would be a good idea to put this as a comment in the code itself to signal to any future devs that they shouldn't reorder this particular section without a lot of testing